scripts/sct_process_segmentation.py

```diff
@@ -22,8 +22,8 @@
 import pandas as pd
 
```

Comment:
i'm also doing quite a lot of modifications on this function currently, so many of these modifications might become obsolete.